### PR TITLE
#12785 fix tests, prevent error when userId is missing.

### PR DESCRIPTION
**Pull request by adi-stefan**
https://github.com/metasfresh/metasfresh/issues/12785

### CODE REVIEW

File: misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/src/test/resources/de/metas/camel/externalsystems/shopware6/happyFlow/50_CamelUpsertBPartnerCompositeRequest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "jsonRequestBPartnerUpsert": {
     "requestItems": [
       {
-        "bpartnerIdentifier": "ext-Shopware6-customerId",
```
Comment (metas-ts) [inline misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/src/test/resources/de/metas/camel/externalsystems/shopware6/happyFlow/50_CamelUpsertBPartnerCompositeRequest.json:6]: please remove the customer-specific name

**Review by metas-ts — CHANGES_REQUESTED** [review]
please check my request and fix the failing unit tests

File: misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/src/test/resources/de/metas/camel/externalsystems/shopware6/happyFlow/50_CamelUpsertBPartnerCompositeRequest.json (hunk 1 above)
Comment (adi-stefan) [reply misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/src/test/resources/de/metas/camel/externalsystems/shopware6/happyFlow/50_CamelUpsertBPartnerCompositeRequest.json:6]: removed.

**Review by metas-ts — COMMENTED** [review]
lgtm

**Pull request merged**